### ISSUE TITLE
Any plans for balanced ternary (simplified) interval arithmetic?

### ISSUE THREAD

**Issue by no-identd** [ThomasTheSpaceFox/BalTcalc#2]
For reference, the IEEE recently released the following simplified subset of IEEE1788-2015, which might help with any such plans, if(f?) any exist:

https://standards.ieee.org/findstds/standard/1788.1-2017.html

**Comment by ThomasTheSpaceFox** [comment]
'BalTcalc' in itself is no longer maintained, and its library, libbaltcalc has been forked into its own repository as part of the SBTCVM project. https://github.com/SBTCVM/libbaltcalc

SBTCVM, for reference, is a python-based Virtual implementation of a made up balanced ternary computer architecture. It also features 2 especially built programming langauges.

If your interested in the latter, id recommend trying SBTCVM Gen 2's SSTNPL, as it has integer variables, a (fairly static) 2-axis table system, and extensive TTY (terminal) io support. Note that only a curses frontend for the VM istelf exists currently, but others are in development.

The assembler on the other hand, is fairly list-like, but does feature importable code files and compile-time variables with a library system. 
repo:
https://github.com/SBTCVM/SBTCVM-Gen2-9
project blog:
https://sbtcvm.blogspot.com/
SSTNPL Guide:
https://sbtcvm.blogspot.com/p/sstnpl-guide.html

**Comment by no-identd** [comment]
>'BalTcalc' in itself is no longer maintained, and its library, libbaltcalc has been forked into its own repository as part of the SBTCVM project. https://github.com/SBTCVM/libbaltcalc

Yes I saw that, but I couldn't open issues there, so I opened one here instead. :) Thanks for the pointers, albeit I had already found all that.

**Comment by ThomasTheSpaceFox** [comment]
Whoops... The issues were turned off. fixed :p
